cli/node/builder.go

```diff
@@ -24,6 +24,7 @@ type cliBuilder struct {
 	daemonFactory DaemonFactory
 	injector      Injector
 	actions       *actionMap
+	enableSignal  bool
```

Comment:
I would explain why we need it, ie. in the case of testing where the notify can cause instability.